lib/auth/auth_with_roles.go

```diff
@@ -1106,6 +1106,16 @@ func (a *ServerWithRoles) GetInventoryStatus(ctx context.Context, req proto.Inve
 	return a.authServer.GetInventoryStatus(ctx, req), nil
 }
 
+// GetInventoryConnectedServiceCounts returns the counts of each connected service seen in the inventory.
+func (a *ServerWithRoles) GetInventoryConnectedServiceCounts() (proto.InventoryConnectedServiceCounts, error) {
+	// only support builtin roles for now, but we'll eventually want to develop an RBAC syntax for
+	// the inventory APIs once they are more developed.
+	if !a.hasBuiltinRole(types.RoleAdmin, types.RoleProxy) {
```

Comment:
Are you planning on doing something with `GetInventoryConnectedServiceCounts` from the proxy?

Reply:
I wasn't, but I left that in there to maintain parity with `InventoryStatus`. I can remove this though.